CHANGELOG.md

```diff
@@ -0,0 +1,210 @@
+# Changelog
+
+## Table of contents
+* [**2.0.0**](#2.0.0)
+* [2.0.1](#2.0.1)
+* [2.0.2](#2.0.3)
+* [2.0.3](#2.0.3)
+* [2.0.4](#2.0.4)
+* [**2.1.0**](#2.1.0)
+* [2.1.1](#2.1.1)
+* [2.1.2](#2.1.2)
+* [2.1.3](#2.1.3)
+* [2.1.5](#2.1.5)
+* [2.1.6](#2.1.6)
+* [2.1.7](#2.1.7)
+* [2.1.8](#2.1.8)
+* [2.1.9](#2.1.9)
+* [**3.0.0**](#3.0.0)
```

Comment:
I see what you are doing (which is fine) but shouldn't the versions be listed in the reverse order?